crm_salesperson_planner_sale/__manifest__.py

```diff
@@ -2,10 +2,10 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
 {
     "name": "Crm Salesperson Planner Sale",
-    "version": "13.0.1.0.0",
+    "version": "15.0.1.0.0",
     "development_status": "Beta",
     "category": "Customer Relationship Management",
-    "author": "Sygel Technology," "Odoo Community Association (OCA)",
+    "author": "Sygel Technology, Tecnativa, Odoo Community Association (OCA)",
```

Comment:
Not sure if these changes qualify for co-authoring. Let's ask @HaraldPanten @ValentinVinagre